### DISCUSSION TITLE
[Feature] skywalking-go add `github.com/lib/pq` support (PostgreSQL)

### DISCUSSION THREAD

**Discussion by fzdwx** [apache/skywalking#12392] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

`skywalking-go` only support `github.com/go-sql-driver/mysql` (MySQL), but i use PostgreSQL.

### Are you willing to submit a pull request to implement this on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
SkyWalking is a volunteer-driven community. If you want to contribute, go ahead. We don't accept feature requests.
